Cookbook/Technical-Documents/PullAssigners.md

```diff
@@ -2,72 +2,64 @@
+   * For the iOS repos, we use the team `@babylonhealth/iOS-Devs`, which should contain every iOS developer at Babylon. So if you need the same list of reviewers, you can just use this one instead of creating one.
+   * If you need a different list to pick your reviewers from, you'd need to create a team via [this page on GitHub](https://github.com/orgs/babylonhealth/new-team), and once it's created, go to the team's page and add Members to that new team ([e.g. here for `iOS-Devs`](https://github.com/orgs/babylonhealth/teams/iOS-Devs/members))
 
-2. Next, **make sure that the proxy team has access to your repository**, via the "Repositories" tab in the team's page (e.g. https://github.com/orgs/babylonhealth/teams/ios-pullassigner/repositories)
+2. Next, **make sure that this GitHub team –containing the people you want as reviewers– has access to your repository**, via the "Repositories" tab in the team's page ([e.g. here for `iOS-Devs`](https://github.com/orgs/babylonhealth/teams/iOS-Devs/repositories))
 
-> _(†) Side note: This proxy team is not strictly necessary, but if you don't use a proxy team and instead use the team you just declared above directly, that means that everybody will be notified and requested review every time a new PR is created. With proxy team Pull Assigners will automatically replace it with only N people from that team requesting review only from them. This is why using a proxy team without any member in it to trigger Pull Assigner is usually preferrable_
+## The `CODEOWNERS` file
 
-### Configure Pull Assigners with those GitHub teams
+Now we need to ensure that the right team (e.g. `iOS-Devs`) is automatically assigned as reviewer on all new PRs.
 
-If you are reusing the `iOS-Admin` and `iOS-PullAssigner` teams for your repo setup, you have nothing special to configure on https://pullreminders.com as those GitHub teams are already configured in Pull Assigner; so you can skip to the next section.
+For this, you just need to create a `.github/CODEOWNERS` file in your repo with the following content (if you have a `CODEOWNERS` file already at the root of your repository, we advise to move it inside a `.github/` directory to clean up your repo root).
 
-But if you have created new teams (team listing reviewers + proxy team) you'll need to log in to https://pullreminders.com and add the GitHub teams to Pull Assigner's configuration via [this screen](https://pullreminders.com/installs/6124714/assigner)
+```
+# Global rule for the whole codebase.
+#  - The empty `iOS-PullAssigner` team will act as a proxy for the PullAssigner bot.
+#    When a new PR arrives, PullAssigner will be triggered, which will then pick members from the iOS-Admin team
```

Comment:
This comment in the `CODEOWNERS` file should be removed as it mentions PullAssigner (and we don't use the concept of a proxy team anymore with GitHub method anyway)

You might want to update the comment to something like this instead
```suggestion
# Global rule for the whole codebase.
# GitHub's Code Reviewer assignment feature will only pick N people from that list
#   to assign them as reviewers. To configure this, go to your Team's page
#   (https://github.com/orgs/babylonhealth/teams/<yourteam>) then to
#   Settings > Code review assignment
```

Reply:
Oops, missed this one (GitHub's search is terrible). Thanks for spotting!